### PR TITLE
Fix most build warnings, clean up projects

### PR DESCRIPTION
**Pull request by idigdoug**
- Add PnpLockdown to all INF files that need them.
- stdunk.h declared inline operator new/delete, but this was a mistake,
  so the declarations are now deprecated and trigger a warning. Fix by
  opting-in to the new system and declaring our own operators.
- Clean up the auto-migrate junk in vcxproj files.
- Replace deprecated ExAllocatePool with ExAllocatePoolWithTag.

Still two build warnings remaining due to use of BootCritical in the INF
files of the SD drivers. Not sure if there is a good way to fix those.

### CODE REVIEW

File: drivers/audio/bcm2836/common.h [hunk 1]
```diff
@@ -346,5 +346,18 @@ NewAdapterCommon
...
+    POOL_TYPE       poolType,
+    ULONG           tag
+);
 
-
+void __cdecl operator delete
+(
+    _Pre_maybenull_ __drv_freesMem(Mem) void* pVoid,
+    _In_ size_t cbSize
+);
```
Comment (mariobalanica) [inline drivers/audio/bcm2836/common.h:363]: It's better/cleaner to have the new/delete operators in their own file, and include the header in `rpiwav.h`.
See: https://github.com/raspberrypi/windows-drivers/pull/3/commits/56501684b1959fdd2b89dd96be6074cae2845163 (it also includes the array delete methods)

This is how Microsoft addressed the issue in their sysvad driver sample (https://github.com/microsoft/Windows-driver-samples/tree/master/audio/sysvad), and

**Review by mariobalanica — CHANGES_REQUESTED** [review]
The property sheet change should be applied to all project files.

File: drivers/RpiLanPropertyChange/bcm2836/RpiLanPropertyChange.vcxproj [hunk 2]
```diff
@@ -22,77 +19,50 @@
...
-  </ImportGroup>
-  <ImportGroup Label="PropertySheets" Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">
-    <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" />
-    <Import Project="$(SolutionDir)\..\bsp.props" />
-  </ImportGroup>
-  <ImportGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64'" Label="PropertySheets">
-    <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" />
-    <Import Project="$(SolutionDir)\..\bsp.props" />
-  </ImportGroup>
-  <PropertyGroup Label="UserMacros" />
```
[the remark this answers is not in the document]
Comment (idigdoug) [reply drivers/RpiLanPropertyChange/bcm2836/RpiLanPropertyChange.vcxproj:93]: Good catch.

File: drivers/audio/bcm2836/common.h (hunk 1 above)
Comment (idigdoug) [reply drivers/audio/bcm2836/common.h:363]: I'm not sure I see any value in defining delete operators that are not used by the component.

I'm not sure I see any value in having a separate header for 2 definitions that should be included everywhere.

If you feel strongly about this, I will change it. I just don't see any benefit to having these in a separate cpp file or a separate header.

Comment (mariobalanica) [reply drivers/audio/bcm2836/common.h:363]: I agree. The other delete operators can be added later on if ever needed.

**Review by mariobalanica — APPROVED** [review]

**Comment by mariobalanica** [comment]
Thanks for the contribution!

**Pull request merged**